packages/grafana-ui/src/components/NodeGraph/NodeGraph.tsx

```diff
@@ -87,8 +87,8 @@ export function NodeGraph({ getLinks, dataFrames, nodeLimit }: Props) {
   // TODO we should be able to allow multiple dataframes for both edges and nodes, could be issue with node ids which in
   //  that case should be unique or figure a way to link edges and nodes dataframes together.
   const processed = useMemo(() => processNodes(nodesDataFrames[0], edgesDataFrames[0]), [
-    nodesDataFrames[0],
-    edgesDataFrames[0],
+    edgesDataFrames,
+    nodesDataFrames,
```

Comment:
Not super important but seems weird this is required if explicitly only first element is used. If eslint cannot figure this out lets do:
```
firstEdgesDataFrame = edgesDataFrames[0]
useMemo(..., [firstEdgesDataFrame, ...])
```